lib/ExecutionEngine/ExecutionEngine.cpp

```diff
@@ -78,21 +78,23 @@ void ExecutionEngine::runBatch(size_t iterations,
   size_t batchSize = vars[0]->getType()->dims()[0];
 
   for (size_t i = 0; i < iterations; i++) {
-    // Launch threads that update the different chunks in the batch:
-    updateForwardBackward(vars, inputs, trainCounter);
+    // Pick up one chunk fron the input and update the minibatch. Then, run a
```

Comment:
I think the first sentence would be more clear this way:
Pick up one slice from the input tensors, and load it into corresponding network Variables.